packages/server/test/hooks.spec.ts

```diff
@@ -234,7 +238,23 @@ describe('Hooks', () => {
         hooks: [
           {
             before: () => {
-              return hookContext;
+              // synchronous hook that doesn't modify context
+            },
+          },
+          {
+            before: () => {
+              return syncHookContext;
+            },
+          },
+          {
+            before: async () => {
+              // asynchronous hook that doesn't modify context
+              await Promise.resolve();
+            },
```

Comment:
Unfortunately, this ran successfully before I made the change to the hook. It looks like our tsconfig ignores test files. I was able to confirm that the change worked with a manual test.

FYI, @toddbaert @lukas-reining

Reply:
I'll see what I can do about that